components/x11/ftgl/manifests/sample-manifest.p5m

```diff
@@ -154,6 +154,7 @@ file path=usr/share/doc/ftgl/html/dir_48342f6a4ef5e4a9839cb593b06f225a.html
 file path=usr/share/doc/ftgl/html/dir_60d10ebdb15d931f338a2058dca6095d.html
 file path=usr/share/doc/ftgl/html/dir_bcba83cc3296faf84eada02145c0eef0.html
 file path=usr/share/doc/ftgl/html/doxygen.css
+file path=usr/share/doc/ftgl/html/doxygen_crawl.html
 file path=usr/share/doc/ftgl/html/faq_8dox.html
```

Comment:
Is there a reason why doxygen_crawl.html is not also added to real the manifest?